projects/1.18-fabric/assets/twigs/twigs/lang/zh_cn.json

```
@@ -0,0 +1,150 @@
+{
+	"mod.twigs.name":"花枝招展",
+	"mod.twigs.contact.discord":"Discord",
+	"itemGroup.twigs.item_group":"花枝招展",
+	"block.twigs.azalea_flowers":"杜鹃花",
+	"block.twigs.twig":"细枝",
+	"block.twigs.pebble":"卵石",

```

```suggestion
	"block.twigs.twig":"树枝",
	"block.twigs.pebble":"石子",
```
这两个是forge版本的翻译，最好统一吧